test/integration/site_layout_test.rb

```diff
@@ -0,0 +1,14 @@
+require "test_helper"
+
+class SiteLayoutTest < ActionDispatch::IntegrationTest
+  test "GET #home renders the about page and checks for hrefs prence" do
+    get root_path
+    assert_template 'static_pages/home'
+    assert_select "a[href=?]", root_path, count: 2
+    assert_select "a[href=?]", help_path
+    assert_select "a[href=?]", about_path
+    assert_select "a[href=?]", contact_path
+    get signup_path
+    assert_select "title", full_title("Signup")
+  end
+end
```

Comment:
These tests dont need anything more now but as you add to the views Id suggest adding more to the tests

Reply:
Where localised copy comes in handy is you can access it in tests meaning you can assert all the expected text is on the page

Reply:
Also a note: DocFox core uses an outdated test strategy / some older patterns. Dont be shocked when you encounter testing you havent seen here ... its very similar and a little different 🤷🏻‍♂️  pair when you first encounter that stuff ... i.e. feature tests and controller tests